schema.graphql

```diff
@@ -93,3 +93,34 @@ type ChangePriceEvent @entity {
   price: Price!
   timestamp: BigInt!
 }
+
+# Types for rewrite. Plan is to merge/replace the old entities when rewrite works.
+
+type PatronNew @entity {
+  id: ID! # Just the address
+  address: Bytes! # address
+  lastUpdated: BigInt!
+  # previouslyOwnedTokens: [Wildcard!]!
+  tokens: [Wildcard!]!
```

Comment:
Should this maybe be tokens: [WildcardNew!]! to reference the new wildcards entity?

Reply:
Hmm, yeah I guess so! It hasn't really mattered until now, but lets commit fully to moving to the new version :)